src/mscorlib/shared/System/TimeZoneInfo.Win32.cs

```diff
@@ -11,6 +11,20 @@
 using System.Threading;
 using Microsoft.Win32;
 
+#if CORERT
```

Comment:
Is that already unified? I thought these are necessary differences between coreclr and corert